src/main/java/com/example/backendspringboottechiteasycontroller/models/Television.java

```diff
@@ -0,0 +1,202 @@
+package com.example.backendspringboottechiteasycontroller.models;
+
+import jakarta.persistence.*;
+
+import java.util.ArrayList;
+import java.util.List;
+@Entity
+@Table(name="cars")
+public class Television {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Integer ID;
+    @Column(name="type")
+    private String type;
+    @Column(name="name")
+    private String name;
+    @Column(name="brand")
+    private String brand;
+    @Column(name="price")
+    private  Double price;
+
+    @Column(name="available_sizes")
+    private Double availableSizes;
+    @Column(name="refresh_rate")
+    private Double refreshRate;
+    @Column(name="screen_type")
+    private String screenType;
+    @Column(name="screen_quality")
+    private String screenQuality;
+    @Column(name="smart_tv")
```

Comment:
This name is actually differen, because Hibernate would translate it into "smart_t_v"